diff --git a/node_modules/vocs/_lib/app/components/Sidebar.js b/node_modules/vocs/_lib/app/components/Sidebar.js
index 203ded0..cc1acc0 100644
--- a/node_modules/vocs/_lib/app/components/Sidebar.js
+++ b/node_modules/vocs/_lib/app/components/Sidebar.js
@@ -48,7 +48,7 @@ function getSidebarGroups(sidebar) {
 }
 function getActiveChildItem(items, pathname) {
     return items.find((item) => {
-        if (matchPath(pathname, item.link ?? ''))
+        if (matchPath(pathname, item.link ?? '') && pathname !== '/')
             return true;
         if (item.link === pathname)
             return true;
@@ -64,7 +64,7 @@ function SidebarItem(props) {
     const match = useMatch(item.link || '');
     const hasActiveChildItem = useMemo(() => (item.items ? Boolean(getActiveChildItem(item.items, pathname)) : false), [item.items, pathname]);
     const [collapsed, setCollapsed] = useState(() => {
-        if (item.link && match)
+        if (item.link && match && pathname !== '/')
             return false;
         if (!item.items)
             return false;
